frontend/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
I think, it must be shorter. 
Also, documentation for project becomes from root.